### ISSUE TITLE
[Screen reader - ADF - Add connection] Screen reader doesn't announce copied status for blob containers.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3854]
**User Experience:**
Screen reader user will be affected if screen reader doesn't announce status of any control as user won't get an idea of current status for such control and will be confused too.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1
Screen reader: Narrator, NVDA

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it  

**Repro Steps:​**
- Launch Azure Storage Explorer app.
- Navigate and select 'Toggle Explorer' at left panel.
- Now, navigate to any of the blob containers and copy that blob containers.
- Verify copied blob containers.

**Actual Result:**
Screen reader doesn't announce copied status for blob containers on Toggle explorer window. It only announce the name of blob containers and due to this, screen reader user is unable to get the copied information for such containers. User will confused whether blob containers has been copied or not. 

**Expected Result:**
Screen reader must announce copied status for blob containers on Toggle explorer window. It should announce the name of blob containers along with copied status so that screen reader could get copied status for such blob containers.

**Note:**
- NVDA also doesn't announce copied status for blob containers.
- On the MAC, the voice-over does not announces the copied status.

[MAS1.3.1_Toggle Explorer.zip](https://github.com/microsoft/AzureStorageExplorer/files/5641855/MAS1.3.1_Toggle.Explorer.zip)
